client/settings/components/shipping-services/group.js

```diff
@@ -75,16 +68,15 @@ const ShippingServiceGroup = ( props ) => {
 			expandedSummary={ summary }
 			clickableHeader={ true }
 			compact
-			actionButton={ actionButton }
-			actionButtonExpanded={ actionButton }
+			screenReaderText={ __( 'Expand Services' ) }
 			expanded={ ! _.isEmpty( errors ) }
 		>
-			<div className="wcc-shipping-service-entry multi-select-header">
-				<span className="wcc-shipping-service-header service-name">{ __( 'Service' ) }</span>
-				<span className="wcc-shipping-service-header price-adjustment">
+			<div className="shipping-services__entry multi-select-header">
```

Comment:
Technically no, but ESLint overlooked it because there's a valid class. Or else, things like `something__valid is-error` wouldn't be valid. But I've changed it anyways in https://github.com/Automattic/woocommerce-services/pull/1051/commits/eaf2e2c144cb489f3495731efbe6b11f4d4072c3